src/inspector_agent.cc

```diff
@@ -658,9 +686,21 @@ bool Agent::StartIoThread(bool wait_for_connect) {
   }
 
   v8::Isolate* isolate = parent_env_->isolate();
+  HandleScope handle_scope(isolate);
+
+  // Enable tracking of async stack traces
+  if (!enable_async_hook_function_.IsEmpty()) {
+    Local<Function> enable_fn = enable_async_hook_function_.Get(isolate);
```

Comment:
I'll tak a look at the commit tomorrow, but the function is used in the JS bindings API for inspector (require('inspector').Session). Check its documentation for how to use it. #14231 has a few examples of how to test it.